packages/frontend/src/components/UserSettings/SlackSettingsPanel/SlackSettingsPanel.styles.ts

```diff
@@ -0,0 +1,37 @@
+import { Button } from '@blueprintjs/core';
+import styled from 'styled-components';
+
+export const AppearancePanelWrapper = styled.div`
+    height: 100%;
+    display: flex;
+    flex-direction: column;
+`;
+
+export const Title = styled.h3`
+    margin: 0 0 10px;
+`;
+export const ColorPalette = styled.div`
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr;
+    column-gap: 40px;
+`;
+
+export const ColorSquare = styled.div`
+    height: 30px;
+    width: 30px;
+    padding: 0.286em;
+`;
+
+export const ColorSquareInner = styled.div`
+    height: 100%;
+    width: 100%;
+    border-radius: 1px;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+`;
```

Comment:
I feel like most of the css from this file is not relevant to this page and can be removed
<img width="866" alt="CleanShot 2022-11-23 at 14 33 10@2x" src="https://user-images.githubusercontent.com/962095/203525207-ebc0772f-1fee-443d-8d53-f07bf9808b81.png">